src/extension/content-script/batteries/GeyserProject.ts

```diff
@@ -0,0 +1,45 @@
+import getOriginData from "../originData";
+import { setLightningData } from "./helpers";
+
+const urlMatcher = /^https:\/\/geyser.fund\/([^/]+)(\/([^/]+))?$/;
+
+const battery = (): void => {
+  const urlParts = document.location.pathname.split("/");
+  const project = urlParts[1];
+  const name = urlParts[2];
+
+  if (project && name) {
+    handleProjectPage(name);
+  }
+};
+
+function handleProjectPage(name: string) {
+  const address =
+    document.querySelector<HTMLElement>("#lightning-address")?.innerText;
+  const title =
+    document.querySelector<HTMLElement>("#project-title")?.innerText;
+  const icon = document.querySelector<HTMLImageElement>(
+    "#project-avatar img"
+  )?.src;
+
+  if (address) {
+    setLightningData([
+      {
+        method: "lnurl",
+        address: address,
+        ...getOriginData(),
+        description:
+          "Geyser - global bitcoin crowdfunding that helps builders and creators fund their ideas.",
```

Comment:
do we need this? it's a bit long imp. 
```suggestion
        description: "geyser.fund",
```